packages/labs/src/components/popover/popover2.tsx

```diff
@@ -76,7 +76,14 @@ export interface IPopover2Props extends IOverlayableProps, IProps {
      * Prevents the popover from appearing when `true`.
      * @default false
      */
-    isDisabled?: boolean;
+    disabled?: boolean;
+
+    /**
+     * Whether the popover is rendered inline (as a sibling of the target element).
+     * If false, it is attached to a new element appended to `<body>`.
+     * @default false
+     */
+    inline?: boolean;
 
```

Comment:
@llorca yep, this prop is already available in the props interface due to the fact that `IPopoverProps2 extends IOverlayableProps`. Note in `overlay.tsx#L49` that `IOverlayableProps` already includes `inline?: boolean`, so we should delete this re-definition here.